docs/artifacts/how-wide-should-the-guard-go.html

```
@@ -0,0 +1,93 @@
+<title>How Wide Should The Guard Go</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)